R/netzschleuder.R

```diff
@@ -210,8 +228,12 @@ ns_df <- function(name, token = NULL, size_limit = 1) {
     nodes_df[["y"]] <- mat[2, ]
   }
 
-  gprops_df <- readLines(unz(temp, gprops_file_name))
+  con_gprops <- unz(temp, gprops_file_name)
+  gprops_df <- readLines(con_gprops)
 
+  on.exit(close(con_edge))
+  on.exit(close(con_nodes))
```

Comment:
All these `on.exit()` calls belong just after the connection is opened, to guard against premature exits.